torchreid/integration/sc/parameters.py

```diff
@@ -56,6 +70,34 @@ class __LearningParameters(ParameterGroup):
             affects_outcome_of=ModelLifecycle.TRAINING
         )
 
+    @attrs
+    class __NNCFOptimization(ParameterGroup):
+        header = string_attribute("Optimization by NNCF")
+        description = header
+
+        enable_quantization = configurable_boolean(
+            default_value=True,
+            header="Enable quantization algorithm",
+            description="Enable quantization algorithm",
+            affects_outcome_of=ModelLifecycle.TRAINING
+        )
+
+        enable_pruning = configurable_boolean(
+            default_value=False,
+            header="Enable filter pruning algorithm",
+            description="Enable filter pruning algorithm",
+            affects_outcome_of=ModelLifecycle.TRAINING
+        )
+
+        maximal_accuracy_degradation = configurable_float(
+            default_value=1.0,
```

Comment:
0.1

Reply:
Thanks for the comment! I changed the default value in the templates.